### ISSUE TITLE
"Close" button of off-canvas navigation is not labelled

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#1650]

**Comment by marcus-herrmann** [comment]
Related with https://github.com/owncloud/owncloud-design-system/pull/497

**Comment by LukasHirt** [comment]
We no longer have off-canvas navigation.
